### PR TITLE
Adding Controlling Endpoints

### PR DESCRIPTION
**Pull request by jamesjahng**
Adding WSUS faq
Adding Inspecting TCP Connections (also fixed toctree)
Adding Checking Password Validation (also fixed toctree)

### CODE REVIEW

File: source/endpoints/auth-agent.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Authenticate User Using Genian Agent
+====================================
+
+Policy Server communicates with the Agent to collect TCP Connection Information periodically and disables a network interface that exceeds the configured connection limits.
+
+See the Configuration of the Plugin
+-----------------------------------
+
+- Go to **System > Update > Genian Software > Plugin > Authenticate User Using Genian Agent**
```
Comment (weatonusmc) [inline source/endpoints/auth-agent.rst]: KyeYeon no longer wants this section in the Admin Guide. Remove "See the Configuration of the Plugin"
You can also help me to remove the others in other pages as well.

File: source/endpoints/chk-pwdvalid.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Checking Password Validation
+============================
+
+Policy Server communicates with the Agent to collect TCP Connection Information periodically and disables a network interface that exceeds the configured connection limits.
+
+See the Configuration of the Plugin
+-----------------------------------
+
+- Go to **System > Update > Genian Software > Plugin > Checking Password Validation**
```
Comment (weatonusmc) [inline source/endpoints/chk-pwdvalid.rst]: KyeYeon no longer wants this section in the Admin Guide. Remove "See the Configuration of the Plugin"
You can also help me to remove the others in other pages as well.

File: source/endpoints/ctrl-traffic.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
+Controlling Network Traffic
+===========================
+
+Policy Server communicates with the Agent to collect TCP Connection Information periodically and disables a network interface that exceeds the configured connection limits.
+
+See the Configuration of the Plugin
+-----------------------------------
+
+- Go to **System > Update > Genian Software > Plugin > Controlling Network Traffic**
```
Comment (weatonusmc) [inline source/endpoints/ctrl-traffic.rst]: KyeYeon no longer wants this section in the Admin Guide. Remove "See the Configuration of the Plugin"
You can also help me to remove the others in other pages as well.

File: source/endpoints/inspect-tcpconn.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
+Inspecting TCP Connections
+==========================
+
+Policy Server communicates with the Agent to collect TCP Connection Information periodically and disables a network interface that exceeds the configured connection limits.
+
+See the Configuration of the Plugin
+-----------------------------------
+
+- Go to **System > Update > Genian Software > Plugin > Inspect TCP Connections**
```
Comment (weatonusmc) [inline source/endpoints/inspect-tcpconn.rst]: KyeYeon no longer wants this section in the Admin Guide. Remove "See the Configuration of the Plugin"
You can also help me to remove the others in other pages as well.

File: source/faq.rst [hunk 5]
```diff
@@ -31,3 +31,8 @@ What is the difference between Windows and macOS agents?
 
 See :doc:`compare-agent-by-os`
 
+How can I check Windows update of endpoints?
+--------------------------------------------
+
+See step 1 on :doc:`endpoints/windows-update`
```
Comment (weatonusmc) [inline source/faq.rst]: Remove "See". Just use Step 1. blah, blah, blah

File: source/sphinx2.rst [hunk 6]
```diff
@@ -108,6 +108,30 @@ Editing contents using Sublime Text
  <Space Needed>
   Lines of Code with no spaces to follow (*Single space before "Lines" needed)
   
+Build your RST file to HTML
+---------------------------
+**Install Sphinx on you PC**
```
Comment (weatonusmc) [inline source/sphinx2.rst]: Install Sphinx on "your" PC

**Comment by jamesjahng** [comment]
Thanks for your comments @weatonusmc 
I will fix it

**Comment by jamesjahng** [comment]
In my 5.0.14R server, The Agent Action numbers are win=34, mac=7, total=41
Am I right?

File: source/sphinx2.rst (hunk 6 above)
Comment (jamesjahng) [reply source/sphinx2.rst]: clear

File: source/faq.rst (hunk 5 above)
Comment (jamesjahng) [reply source/faq.rst]: clear

File: source/endpoints/inspect-tcpconn.rst (hunk 4 above)
Comment (jamesjahng) [reply source/endpoints/inspect-tcpconn.rst]: I will remove that section of all endpoint contents

Comment (jamesjahng) [reply source/endpoints/inspect-tcpconn.rst]: clear
1. Removed "See the Configuration of the Plugin"
2. Changed "see" -> "check" in antivirus part

**Pull request closed without merging**